aurora/src/main/java/digital/murl/aurora/effects/Action.java

```diff
@@ -0,0 +1,38 @@
+package digital.murl.aurora.effects;
+
+import digital.murl.aurora.Result;
+
+import java.util.Map;
+
+@FunctionalInterface
+public interface Action<T extends Effect, H extends Map<String, Object>> {
```

Comment:
Action is ambiguous, should both be renamed AgentAction and EffectAction ?

Reply:
Since both systems use the same format and might be interchangable in some specific cases I think both using the same name is fine.

`extends Effect` should be changed so it can be either Agent or Effect

Reply:
should be resolved by [27d3eca](https://github.com/murl-digital/aurora/pull/44/commits/27d3ecae2cafd21b4e27e17c3eee33abb089b845), im not going to keep them seperate but with more specific names to avoid confusion